lib-multisrc/peachscan/src/eu/kanade/tachiyomi/multisrc/peachscan/PeachScan.kt

```diff
@@ -82,6 +84,18 @@ abstract class PeachScan(
 
     override fun latestUpdatesNextPageSelector() = null
 
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
+        if (query.startsWith(URL_SEARCH_PREFIX)) {
+            val manga = SManga.create().apply { url = "/${query.substringAfter(URL_SEARCH_PREFIX)}" }
```

Comment:
```suggestion
            val manga = SManga.create().apply { url = "/${query.substringAfter(URL_SEARCH_PREFIX)}/" }
```
avoid 302 redirect

Reply:
I noticed that too, I think it was removed in a refactoring 😅